src/views/QuestionnaireEdit/SelectMultipleQuestionsForm/index.tsx

```diff
@@ -0,0 +1,256 @@
+import { useCallback, useMemo } from 'react';
+import {
+    isNotDefined,
+    randomString,
+} from '@togglecorp/fujs';
+import {
+    Button,
+    SelectInput,
+    TextInput,
+    useAlert,
+} from '@the-deep/deep-ui';
+import { gql, useMutation, useQuery } from '@apollo/client';
+import {
+    ObjectSchema,
+    createSubmitHandler,
+    requiredStringCondition,
+    getErrorObject,
+    useForm,
+    PartialForm,
+} from '@togglecorp/toggle-form';
+
+import {
+    CreateTextQuestionMutation,
+    CreateTextQuestionMutationVariables,
+    PillarsQuery,
+    PillarsQueryVariables,
+    QuestionCreateInput,
+    QuestionTypeEnum,
+} from '#generated/types';
+
+import styles from './index.module.css';
+import SelectMultipleQuestionPreview from '#components/SelectMultipleQuestionsPreview';
+
+const CREATE_TEXT_QUESTION = gql`
```

Comment:
The query should be named something else like `CREATE_MULTIPLE_SELECTION_QUESTION`. Let's not use misleading names.